src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/NocRespondentRepresentativeService.java

```diff
@@ -135,7 +143,21 @@ public void updateRepresentativesAccess(CallbackRequest callbackRequest) throws
             log.info("About to apply representation change {}", changeRequest);
 
             nocCcdService.updateCaseRepresentation(accessToken, changeRequest,
-                caseDetails.getJurisdiction(), caseDetails.getCaseTypeId(), caseDetails.getCaseId());
+                    caseDetails.getJurisdiction(), caseDetails.getCaseTypeId(), caseDetails.getCaseId());
+
+            String email = caseDataBefore.getRepCollection()
+                    .stream().filter(r -> r.getValue().getRespondentOrganisation().getOrganisationID()
+                            .equals(changeRequest.getOrganisationToRemove().getOrganisationID()))
```

Comment:
Is the spacing correct on the .equals line

could we put changeRequest.getOrganisationToRemove().getOrganisationID() in a variable?

Reply:
Spacing looks to be correct, and yes refactored to use new variable